src/utils.js

```diff
@@ -148,3 +148,49 @@ export function sanityCheck(
 
   return null;
 }
+
+export function getCryptoPassword(password: string) {
+  return `{SHA}'${crypto
+    .createHash('sha1')
+    .update(password, 'binary')
+    .digest('base64')}`;
+}
+
+/**
+ * changePasswordToHTPasswd - change password for existing user
+ * @param {string} body
+ * @param {string} user
+ * @param {string} passwd
+ * @param {string} newPasswd
+ * @returns {string}
+ */
+export function changePasswordToHTPasswd(
+  body: string,
+  user: string,
+  passwd: string,
+  newPasswd: string
+): string {
+  if (crypt3) {
+    passwd = crypt3(passwd);
```

Comment:
According to FP Programming, we should not reassign the parameters ... but ... after reading this chapter of the book 'Functional Light...'  https://github.com/getify/Functional-Light-JS/blob/master/manuscript/ch5.md/#chapter-5-reducing-side-effects . This is something that should be analyzed. So...in this case...I think it's fine.